stories/topics/con-saas-intro.adoc

```diff
@@ -1,14 +1,9 @@
 [id="con-saas-intro"]
 = Introduction 
-{SaaSonAWS} links your {AWS} (AWS) offering to your Red{nbsp}Hat account which enables access to {PlatformNameShort}. 
-When you subscribe to {SaaSonAWSShort}, Red{nbsp}Hat deploys and configures an  {PlatformNameShort} environment for you. 
+The Red{nbsp}Hat Ansible Automation Platform Service on Amazon Web Services (AWS) provides you with an {PlatformNameShort} deployment purchased through AWS Marketplace. 
```

Comment:
I've intentionally not used the attribute here so I can spell out what AWS is then just refer to the acronym through the section.